solrmonitor/solrmonitor.go

```diff
@@ -655,6 +691,7 @@ func parseStateData(name string, data []byte, version int32) (*CollectionState,
 }
 
 func (coll *collection) start() error {
+	coll.isWatched = false
```

Comment:
Do we need synchronization here? When `isWatched` is set to true in `collection.WatchAdded`, we acquire the lock first.